tests/integration_python/test_run_cli.py

```diff
@@ -372,3 +377,101 @@ def test_run_dry_run(pixi: Path, tmp_pixi_workspace: Path) -> None:
         stdout_excludes="WET",
         stderr_excludes="WET",
     )
+
+
+def test_run_args(pixi: Path, tmp_pixi_workspace: Path) -> None:
+    manifest = tmp_pixi_workspace.joinpath("pixi.toml")
+    toml = f"""
+    {EMPTY_BOILERPLATE_PROJECT}
+    [tasks]
+    """
+    manifest.write_text(toml)
+
+
+def test_invalid_task_args(pixi: Path, tmp_pixi_workspace: Path) -> None:
+    manifest = tmp_pixi_workspace.joinpath("pixi.toml")
+    toml = f"""
+    {EMPTY_BOILERPLATE_PROJECT}
+    [tasks]
+    task_invalid_defaults = {{ cmd = "echo Invalid defaults: {{{{ arg1 }}}} {{{{ arg2 }}}} {{{{ arg3 }}}}", args = [{{ name = "arg1", default = "default1" }}, "arg2", {{ name = "arg3", default = "default3" }}] }}
```

Comment:
```suggestion
    task_invalid_defaults = {{ cmd = "echo Invalid defaults: {{{{ arg1 }}}} {{{{ arg2 }}}} {{{{ arg3 }}}}", args = [{{ name = "arg1", default = "default1" }}, "arg2", {{ name = "arg3", default = "default3" }}] }}
```
I think at that point it is easier to read the toml as a dict, add entries and serialize it again